allennlp/commands/train.py

```diff
@@ -108,7 +109,10 @@ def train_model(param_dict: Dict[str, Any]):
     params = Params(replace_none(param_dict))
     prepare_environment(params)
 
-    log_dir = params.get("serialization_prefix", None)  # pylint: disable=no-member
+    trainer_params = params.pop("trainer")
+    log_dir = trainer_params.get("serialization_prefix")
```

Comment:
Hmm, looks like I broke this with a recent PR without realizing it.  I was wondering why the logging output of my recent BiDAF training run wasn't saved like I expected.  Is there some way we can test this against current example experiment configs?  It would have been harder for me to miss the fact that I broke this.  I'm not sure what test would have caught this particular error, though...